custom_components/tuya_local/devices/caldo_up_t_wifi.yaml

```diff
@@ -4,20 +4,18 @@ products:
     name: Olimpia Splendid Caldo Up T
 primary_entity:
   entity: climate
-  translation_only_key: heater
   dps:
     - id: 1
       type: boolean
       name: hvac_mode
       mapping:
         - dps_val: true
-          value: heat
+          value: "heat_cool"
         - dps_val: false
           value: "off"
     - id: 2
       type: integer
       name: temperature
-      unit: C
       range:
```

Comment:
Removing the unit is not just without reason, it breaks the integration on HA 2024.6 and later, which requires the unit to be set.